scripter.py

```diff
@@ -43,6 +43,17 @@ def __linux_create_wallpaper_script(pokemon):
     if "gnome" in os.environ.get("GDMSESSION"):
         fmt = 'gsettings set org.gnome.desktop.background picture-uri "file://{}"'
         return fmt.format(pokemon.get_path())
+    
+    if "xubuntu" in os.environ.get("GDMSESSION"):
```

Comment:
What's is your value of `$DESKTOP_SESSION` I've done some research and that seems to be a more global variable. Try using that instead if it's something like XFCE